examples/gno.land/p/demo/memeland/memeland.gno

```diff
@@ -238,3 +238,7 @@ func (p PostSorter) Swap(i, j int) {
 func (p PostSorter) Less(i, j int) bool {
 	return p.LessF(i, j)
 }
+
+func callerIsOwner(addr std.Address) bool {
+	return std.PrevRealm().Addr() == addr
+}
```

Comment:
this is still not doing what the title is suggesting.
there is no notion of "owner".

Reply:
https://github.com/gnolang/gno/pull/2198/commits/8bd3ac0bd411c9f2adee3bbc466e57ed6f33ac5b